packages/devtools/cli/src/util/util.ts

```diff
@@ -84,3 +85,35 @@ export const printMembers = (members: SpaceMember[], flags = {}) => {
     }
   );
 };
+
+export const mapCredentials = (credentials: Credential[], truncateKeys = false) => {
```

Comment:
Refactor util file to co-locate all "print" related methods (always factor out "util" as it grows)

Reply:
+1